addons/event/views/event_registration_views.xml

```diff
@@ -17,7 +17,7 @@
                 <field name="phone" optional="show"/>
                 <field name="company_name" optional="hide"/>
                 <field name="event_id" column_invisible="context.get('default_event_id')" readonly="state != 'draft'"/>
```

Comment:
Hmm, :thinking: 
`column_invisible="context.get('default_event_id')"` with this I don't think it will visible in first place, so I did not remove readonly from there.
If you want we can remove it from here as well :smiley:

Reply:
I don't think the context influences the readonly feature, so yes it would be best to align IMO 🙂

Reply:
Updated :smile: